app/_src/gateway-operator/guides/konnect-entities/cloud-gateways-data-plane-group-configuration.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Cloud Gateways Data Plane Group Configurations
+---
+
+{% assign entity = "Data Plane Group Configuration" %}
+{% assign crd = "KonnectCloudGatewayDataPlaneGroupConfiguration" %}
+
+In this guide you'll learn how to use the `{{ crd }}` custom resource to
+manage {{site.konnect_product_name}} Dedicated Cloud Gateways {{ entity }}s natively from your Kubernetes cluster.
+
+{% include md/kgo/konnect-entities-prerequisites.md disable_accordian=false version=page.version release=page.release
+with-control-plane=true %}
+
+{% include md/kgo/konnect-cloud-gateways-provider-account.md entity='Data Plane Group Configuration' %}
+
+## Create a Cloud Gateway {{ entity }}
+
+Creating the `{{ crd }}` object in your Kubernetes cluster will provision a new {{site.konnect_short_name }} Dedicated Cloud Gateway {{ entity }}.
+
+You can refer to the [`{{ crd }}` CRD API](/gateway-operator/{{ page.release }}/reference/custom-resources/#konnectcloudgatewaynetwork)
+for all the available fields.
+
+To create a `{{ crd }}` object you can use the following YAML manifest:
+
+```bash
+echo '
+apiVersion: konnect.konghq.com/v1alpha1
+kind: {{ crd }}
+metadata:
+  name: konnect-cg-dpconf
+spec:
+  api_access: private+public
+  version: "3.9"
```

Comment:
Is there an easy way to get this `<major>.<minor>` using variables like `{{ site.data.kong_latest_gateway.ce-version }}`?

I was thinking about:

```
{% assign version_parts = site.data.kong_latest_gateway.ce-version | split: "." %}
{% assign major = version_parts[0] %}
{% assign minor = version_parts[1] %}
{% assign trimmed_version = major | append: "." | append: minor %}
```

but this feels unnecessarily verbose.

Reply:
cc: @Kong/docs-maintainers

Reply:
There's not a good way to do this today. I solved it with:

```
version: "{{ site.data.kong_latest_gateway.ee-version | split: "." | slice: 0,2 | join: "." }}"
```